module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaPrivateAccessApplicationSegment.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Get-EntraBetaPrivateAccessApplicationSegment
+description: This article provides details on the Get-EntraBetaPrivateAccessApplicationSegment command.
+
+ms.topic: reference
+ms.date: 06/26/2024
+ms.author: andresc
```

Comment:
```suggestion
ms.author: eunicewaweru
```

Reply:
Adding a Microsoft author (Content Dev owner). Your name is still in the author tag.